tests/timeseries/test_quantities.py

```diff
@@ -563,8 +626,14 @@ def test_to_and_from_string(
     generation and regenerate it with the more appropriate unit and precision.
     """
     quantity = quantity_type.__new__(quantity_type)
-    # pylint: disable=protected-access
-    quantity._base_value = value * 10**exponent
+    quantity._base_value = value * 10**exponent  # pylint: disable=protected-access
+    # The above should be replaced with:
+    # quantity = quantity_type._new(  # pylint: disable=protected-access
+    #     value, exponent=10**exponent
```

Comment:
I think here we only need to pass the exponent as the method `_new()` will do the math `10.0**exponent`
```suggestion
    #     value, exponent=exponent
```